tutorial/consuming_packages/the_flexibility_of_conanfile_py.rst

```diff
@@ -195,7 +195,7 @@ Use the layout() method
 In the previous examples, every time we executed a `conan install` command we had to use
 the `--output-folder argument` to define where we wanted to create the files that Conan
 generates. Also, note that we used a different folder when building in Windows or in
-Linux/Macos depending if we were using a multi-config CMake generator or not. You can
+Linux/macOS depending if we were using a multi-config CMake generator or not. You can
```

Comment:
``Macos`` is the Conan ``setting`` defined in ``settings.yml``, it is not that bad that it is written that way in the docs.

Reply:
Oh right, I just followed the trademark info that Apple published to have consistent naming